src/TraceEvent/Symbols/NativeSymbolModule.cs

```diff
@@ -107,24 +107,27 @@ public string FindNameForRva(uint rva, ref uint symbolStartRva)
                 ret = ret.Substring(0, dollarIdx);
 
             // See if we have a Project N map that maps $_NN to a pre-merged assembly name 
-            var mergedAssembliesMap = GetMergedAssembliesMap();
-            if (mergedAssembliesMap != null)
+            if (LookupAssemblyNameForCompiledNative)
```

Comment:
I don't really want the LookupAssemblyNameForCompiledNative flag.    In theory only PDBs that come from .NET Native have this section and thus would be affected.   Thus I don't understand why you would need to change this code.

Can you describe more about what fails if this code is in place.   I would hope that what happens is that GetMergedAssemblyMap() returns null and all the code below does not execute, and so you don't care.   

Does that not happen?     

Ideally we don't have to change NativeSymbolModule.cs at all.

Reply:
No, I don't like it more than you do. I didn't have time to look into why symbol match failed in our scenario when these lines of code are on. I'll spend some time to kick the tire again since there have been changes.